README.rst

```diff
@@ -3,46 +3,40 @@ weaklensingclustersbi
 
 This repository contains tools for experiments for Weak Lensing Galaxy Clusters using Simulation Based Inference.
 
-
-
 ---------------
 
 Summary:
 
 We've split our workflow into four parts:
 
-1) Simulation (/configs/simulations) - sim_config tells us about how our simulations were generated/obtained (e.g. the mass-concentration relation used, the log10mass range, the number of simulations, the noise dex, etc).
-2) Observation(/configs/observations) - obs_config tells us about how our observations were generated/obtained (e.g. the richness-mass relation used, the richness range, the number of observations, the noise dex, etc). 
-3) Inference (/configs/inference) - infer_config tells us about how our inference was done (e.g. are we using SBI or MCMC, our priors, etc.)
+1) Simulation (/configs/simulations) - sim_config tells us about how our simulations were generated/obtained (e.g. the mass-concentration relation used, how much scatter the m-c relation has, the log10mass range, etc).
```

Comment:
These are great!  Can you also put together and put in the readme a visualization of the workflow in a pipeline diagram?